internal/models/payout.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "time"
+
+type Payout struct {
+	Timestamp      time.Time `json:"timestamp"`
+	PaymentDetails map[string]NodeStatsDetails
```

Comment:
Why is this map[string]NodeStatsDetails and not just separate NodeID and NodeStatsDetails